bin/mhcflurry_neoepitope_binding_prediction.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python
+
+# Written by Leon Bichmann and released under MIT license.
```

Comment:
Lukas Heumos also contributed to some of these scripts, although I am unaware which ones